src/components/Form.js

```diff
@@ -39,27 +39,32 @@ const Form = () => {
   };
 
```

Comment:
- [ ] @billymohajeri kindly remove the commented code on line 333, its not good practise to have commented code in the codebase.